types/targeting.ts

```diff
@@ -1,12 +1,13 @@
 export interface Audience {
-    name: string;
-    description: string;
-    targeting?: string;
-    filter_expression?: string;
-    /**
-     * A boolean BigQuery SQL expression expressing whether a single row in telemetry.main would have
-     * matched the audience definition. These are used for sizing experiment populations,
-     * not for targeting.
-     */
-    desktop_telemetry?: string;
-  }
\ No newline at end of file
+  name: string;
+  description: string;
+  targeting?: string;
+  filter_expression?: string;
+  /**
+   * A boolean BigQuery SQL expression expressing whether a single row in telemetry.main would have
+   * matched the audience definition. These are used for sizing experiment populations,
+   * not for targeting.
+   */
+  desktop_telemetry?: string;
+  firefox_channel?: string;
```

Comment:
We can be a bit stronger with our versions here:

```suggestion
  firefox_channel?: 'Nightly' | 'Beta' | 'Release';
```